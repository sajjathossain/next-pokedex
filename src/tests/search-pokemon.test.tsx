import SearchPokemon from 'components/search-pokemon';
import { render } from '@testing-library/react';

const renderComponent = () => {
  return render(<SearchPokemon />);
};

describe('Search Pokemon Component', () => {
  it('checks if search pokemon component renders perfectly', () => {
    const { container } = renderComponent();
    expect(container).toMatchSnapshot();
  });
});