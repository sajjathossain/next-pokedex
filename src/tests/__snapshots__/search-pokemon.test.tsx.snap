// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Pokemon Component checks if search pokemon component renders perfectly 1`] = `
<div>
  <div
    class="w-3/4 mx-auto flex justify-center mt-4 bg-transparent"
  >
    <div
      class="flex rounded-full px-4 w-2/4 max-h-max bg-gray-50 focus-within:ring-2 ring-blue-600"
    >
      <input
        autocomplete="none"
        class="ring-0 outline-none border-0 h-10 bg-transparent flex flex-1"
        placeholder="Enter any pokemon name*"
        required=""
        type="text"
        value=""
      />
      <button
        class="w-auto pointer-events-none"
      >
        <svg
          class="h-6 w-6 focus-within:text-blue-600 text-gray-500"
          fill="none"
          stroke="currentColor"
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
          />
        </svg>
      </button>
    </div>
  </div>
</div>
`;
